WordPress/Classes/ViewRelated/Cells/InlineEditableMultiLineCell.swift

```diff
@@ -1,17 +1,23 @@
 import Foundation
 
-protocol EditCommentMultiLineCellDelegate: AnyObject {
+// UITableViewCell that displays an editable UITextView to allow text to be modified inline.
+// The cell height resizes as the text is modified.
+// The delegate is notified when:
+// - The height is updated.
+// - The text is updated.
+
+protocol InlineEditableMultiLineCellDelegate: AnyObject {
     func textViewHeightUpdated()
     func textUpdated(_ updatedText: String?)
```

Comment:
Just one minor question. Should we also add a `cell` parameter here, in case if there are multiple multi-line cells? Similar to the protocol method of `InlineEditableSingleLineCellDelegate`. E.g.:

```Swift
func textViewHeightUpdated(_ cell: InlineEditableMultiLineCell)
func textUpdated(_ cell: InlineEditableMultiLineCell, updatedText: String?)
```

Reply:
> in case if there are multiple multi-line cells?

I'd hate to see _that_ view. It would just be horrible. 😬  As ill-advised as that is, I made the change for consistency. Enjoy your multi multiline view. 😉